compiler/rustc_error_codes/src/error_codes/E0464.md

```diff
@@ -0,0 +1,11 @@
+The compiler was unable to determine which version of a crate dependency should
+be used, due to multiple potential versions being available.
+
+For example, this error can occur if you import a crate named `dependency`, but
+you specify multiple crates with the name `dependency` in your `Cargo.toml`. Or,
+if you're not using Cargo, this error can occur if you pass multiple `--extern`
+flags for different versions of a crate library file.
```

Comment:
Is this part correct? I feel unsure about when exactly this error occurs and what the cause is. \[Which is why I'm adding an explanation ;)\]

Reply:
Would the following be a more accurate description? Let me know if anything is incorrect in the current version or the suggested version :)

```suggestion
The compiler found multiple library files with the requested crate name.

For example, this error can occur if you import a crate named `dependency`, but
you specify multiple crates with the name `dependency` in your `Cargo.toml`. Or,
if you're not using Cargo, this error can occur if you pass multiple `--extern`
flags where the library files have the same crate name (not the same as the
filename).
```